src/Microsoft.AspNetCore.Mvc.RazorPages/Internal/PageActionInvokerCacheEntry.cs

```diff
@@ -40,14 +43,16 @@ public class PageActionInvokerCacheEntry
         public Func<PageContext, object> ModelFactory { get; }
 
         /// <summary>
-        /// Gets the applicable PageStarts.
+        /// The action invoked to release a model. This may be <c>null</c>.
```

Comment:
It would be cooler to say 'delegate' here so you're not overloading the term action.